Core/src/IfcGeometryConverter/PlacementConverter.h

```diff
@@ -776,10 +795,23 @@ namespace OpenInfraPlatform {
 					return axis;
 				}
 
+				/**********************************************************************************************/
+				/*! \brief Convert \c IfcVirtualGridIntersection to a transformation matrix
+				* \param[in] intersection			The \c IfcVirtualGridIntersection to be converted
+				* \return location					Location of the grid.
+				* \return intersectingAxes1			Tangent of the first intersecting axis	

```

Comment:
```suggestion
				* \return 					A tuple: 1: Location of the grid, 2: tangent of the first intersecting axis	
```

a `\return` cannot be named (or can it?), so the return needs to be described in one line

Reply:
Resolved: 3390ed1